lib/sparse/src/index/search_context.rs

```diff
@@ -55,14 +55,16 @@ impl<'a> SearchContext<'a> {
         }
     }
 
-    /// Score the query vector against the given ids.
-    ///
-    /// This is a plain search without any pruning.
+    /// This is a plain search against the given ids without any pruning.
     ///
     /// The results are not sorted.
```

Comment:
It seems the results are sorted now ;)

Reply:
Sorted by score =) Changed also tests to check sorting of the scoring result. Now it works like regular search and returns top-nearest. Updated comment